client/my-sites/customer-home/cards/education/affiliates/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useHasEnTranslation } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import affiliatesIllustration from 'calypso/assets/images/customer-home/illustration--affiliates-icons-small.png';
+import EducationalContent from '../educational-content';
+export const EDUCATION_EARN = 'home-education-earn';
+
+const EducationAffiliates = () => {
+	const translate = useTranslate();
+	const hasTranslation = useHasEnTranslation();
+
+	const title = translate( 'Earn with the Automattic Affiliate Program' );
+	const hasTitleTranaslation = hasTranslation( 'Earn with the Automattic Affiliate Program' );
+
+	const description = translate(
+		'Join the Automattic Affiliate Program and get up to 100% payouts by promoting one or more of Automattic’s trusted, profitable products.'
+	);
+	const hasDescriptionTranslation = hasTranslation(
+		'Join the Automattic Affiliate Program and get up to 100% payouts by promoting one or more of Automattic’s trusted, profitable products.'
+	);
+
+	if ( ! hasTitleTranaslation || ! hasDescriptionTranslation ) {
+		EducationAffiliates.isDisabled = true;
+		return null;
+	}
+	return (
+		<EducationalContent
+			title={ title }
+			description={ description }
+			links={ [
+				{
+					calypsoLink: true,
+					url: `https://app.impact.com/campaign-campaign-info-v2/Automattic-Inc.brand?ref=wordpressdotcom`,
```

Comment:
I have two sets of comments for this one:
* Do we want a more specific or additional `ref` value to track that we're coming from this card? (This may not be possible, but I wanted to flag it regardless.)
* This will render as an inline link in the current window -- I think we should specify `externalLink: true` to correctly flag it as an external link and open the link in a new window.
```suggestion
					externalLink: true,
					url: `https://app.impact.com/campaign-campaign-info-v2/Automattic-Inc.brand?ref=wordpressdotcom`,
```


I also have a broader concern that it feels like we're always showing these educational cards in the same order every time, which means that many users won't see any secondary items unless they specifically click through the list. I think we should add some randomisation of the order for these cards on the server, so we don't always show the same cards. (I made similar changes for the main cards a year or two ago for very similar reasons.)

Reply:
> Do we want a more specific or additional ref value to track that we're coming from this card?

I'm not sure, but I brought this up in https://github.com/Automattic/Dotcom-Creative-Requests/issues/172#issuecomment-2149436544

> This will render as an inline link in the current window -- I think we should specify externalLink: true to correctly flag it as an external link and open the link in a new window.

Nice catch! Thanks!

> I also have a broader concern that it feels like we're always showing these educational cards in the same order every time, which means that many users won't see any secondary items unless they specifically click through the list.

The plan is to use the first placement only until the end of July (peRNfl-IS-p2#comment-914).
I think I saw something related to randomizing the order but I haven't seen it working yet.

Reply:
I started work on the randomisation in D151586-code. We can discuss whether that should be delayed, or whether we want to keep them in the first slot for all users. I would lean towards randomising them so we get a better comparison RE their interest/utility for folks.

Reply:
@daledupreez Thanks for looking into this!

> I would lean towards randomising them so we get a better comparison RE their interest/utility for folks.

This is a valid point, but I think the intent is to promote these for a short period, then have them be randomized with the rest, or moved to a more appropriate position. In this case, I would suggest delaying this to the end of July (or exclude the affiliates card from the shuffle)
Nevertheless, feel free to post this to the project's feedback thread: peRNfl-Lk-p2